generators/python/src/fern_python/generators/sdk/client_generator/base_wrapped_client_generator.py

```diff
@@ -88,6 +89,9 @@ def _create_wrapper_method(
             )
 
         # Use EndpointFunctionGenerator to create the wrapper method
+        if self._package.service is None:
+            raise ValueError("Package must have a service to generate endpoints")
```

Comment:
This was one place I couldn't figure out how to guarantee the narrowed / required type